public/switch.html

```diff
@@ -205,6 +205,76 @@ <h2 class="step-title">2</h2>
             </svg>
           </div>
         </section>
+        <section class="step" id="full-extra-USB">
+          <div class="step-half step-text">
+            <h2 class="step-title">Extra: USB</h2>
+            <p>
+              You can transfer your Switch screenshots to a computer via a USB
+              cable:
+            </p>
+            <ol>
+              <li>
+                From the Switch's home menu, select <strong>Settings</strong>,
+                <strong>Data Management</strong>,
+                <strong>Manage Screenshots &amp; Videos</strong>
+              </li>
+              <li>
+                Select <strong>Copy to a Computer via USB Connection</strong>
+              </li>
+              <li>
+                Connect your Switch <em>directly</em> to a computer (not via a
+                dock)
+              </li>
+              <li>Choose which screenshots to transfer</li>
+            </ol>
+            <p>
+              For further details, see
+              <a
+                href="https://en-americas-support.nintendo.com/app/answers/detail/a_id/53664/"
+                >Nintendo's documentation</a
+              >. It is also possible to transfer screenshots by using a microSD
+              memory card.
+            </p>
+          </div>
+          <div class="step-half step-diagram">
+            <svg viewBox="0 0 475.2 228.1">
```

Comment:
Good old copy/paste 😃 Good idea to add the symbol in.

This is more a note for me to get back into this project at some point, but I've definitely been avoiding making new images since the process is time consuming and I'm not a designer at all.

Reply:
Ha, yeah. I spent some time trying to tweak a CC0 "wireless connection" icon to look good next to the smart-device instructions but bailed after I realized minor tweaks to the SVG markup by hand wouldn't help.